packages/node_modules/@webex/internal-plugin-encryption/test/integration/spec/kms.js

```diff
@@ -314,6 +316,66 @@ describe('Encryption', function () {
         }));
     });
 
+    describe('upload customer master key 1', () => {
+      let uploadedkeyId;
+
+      it('upload customer master key', () => window.crypto.subtle.generateKey(
+        {
+          name: 'RSA-OAEP',
+          modulusLength: 4096,
+          publicExponent: new Uint8Array([1, 0, 1]),
+          hash: 'SHA-256'
+        },
+        true,
+        ['encrypt', 'decrypt']
+      )
+        .then((keyPair) => {
+          console.warn('get the public key', keyPair.publicKey);
+          const buf = window.crypto.getRandomValues(new Uint8Array(16));
+
+          console.warn(`---the random master key is ${new TextDecoder('utf-8').decode(buf)}`);
+          console.warn(`---the test user org id is ${spock.orgId}`);
+
+          return window.crypto.subtle.encrypt(
+            {
+              name: 'RSA-OAEP'
+            },
+            keyPair.publicKey,
+            buf
+          );
+        })
+        .then((encryptedData) => webex.internal.encryption.kms.uploadCustomerMasterKey({assignedOrgId: spock.orgId, customerMasterKey: new TextDecoder('utf-8').decode(encryptedData)}))
+        .then((uploadRes) => {
+          uploadedkeyId = uploadRes.uri;
+
+          return webex.internal.encryption.kms.listAllCustomerMasterKey({assignedOrgId: spock.orgId});
+        })
+        .then((listCmksRes) => {
+          const cmks = listCmksRes.customerMasterKeys;
+          const uploadedCmk = cmks.find((cmk) => cmk.uri === uploadedkeyId);
+
+          expect(uploadedCmk).to.not.be.null;
+        })
+        .then((uploadedCmk) => webex.internal.encryption.kms.activateCustomerMasterKey({keyId: uploadedCmk.uri, assignedOrgId: spock.orgId}))
+        .then((activeRes) => {
+          expect(activeRes.customerMasterKeys[0].usageState).to.have.string('');
+        })
+        .then(() => webex.internal.encryption.kms.deleteCustomerMasterKey({keyId: uploadedkeyId, assignedOrgId: spock.orgId})));
+    });
+
+    describe('#fetchPublicKey()', () => {
+      it('requests public key from the KMS', () => webex.internal.encryption.kms.fetchPublicKey({assignedOrgId: spock.orgId})
+        .then((publicKey) => {
+          console.warn(`in test, the public key is ${JSON.stringify(publicKey)}`);
```

Comment:
dont need console logs in test unless for debugging

Reply:
Thanks, will remove this finally.